tools/code_format/check_format_test_helper.py

```diff
@@ -232,6 +232,12 @@ def runChecks():
       "Registry::InjectFactory instead.")
   errors += checkUnfixableError("strerror.cc",
                                 "Don't use strerror; use Envoy::errorDetails instead")
+  errors += checkUnfixableError(
+      "std_unordered_map.cc", "Don't use std::unordered_map; use absl::flat_hash_map instead " +
+      "or absl::node_hash_map if pointer stability of keys/values is required")
+  errors += checkUnfixableError(
+      "std_unordered_set.cc", "Don't use std::unordered_set; use absl::flat_hash_set instead " +
+      "or absl::node_hash_set if pointer stability of keys/values is required")
```

Comment:
Is it too ambitious to enable these lint checks in the same PR as the large scale change above?  It may be good to do some smaller PRs which can be reviewed quickly to adjust the code base in preparation for these linter changes.

Reply:
Risky in the sense that existing PRs may be introducing new instances of stdlib unordered containers and depending on merge order master will be broken?

Reply:
You need to be merging constantly to get this large change to land.  Adding this lint checks in a smaller, faster to review PR will likely reduce the difficulty of getting the change merged in.

Reply:
That would prevent this commit from potentially catching newly introduced violations of this rule at the time we adopt the change. Better that we do merge, and correct the newly introduced exceptions

Reply:
I guess the rate at which we add new uses of ordered maps is not high enough to be a problem, so it may be fine to start enforcing immediately. I think the open comment that remains is the one about exception safety.

Reply:
ok sweet, gonna resolve this comment and we can focus on the exception safety solution